apps/web/src/views/BuyCrypto/components/AccordianDropdown/Accordian.tsx

```diff
@@ -16,6 +17,63 @@ function Accordion({
 }) {
   const { t } = useTranslation()
   const [currentIdx, setCurrentIdx] = useState<number | string>(0)
+  const [sortedArray, setSortedArray] = useState([])
+
+  const MoonapyAmt = useCallback(
```

Comment:
For variable naming convention, we use camel case format and normally we don't capitalize the first letter